src/components/coin-row/ListCoinRow.js

```diff
@@ -1,13 +1,13 @@
-import React, { Fragment, useCallback } from 'react';
+import React, { Fragment, useCallback, useMemo } from 'react';
```

Comment:
this useMemo was entirely missing my type checker noticed it but not sure how it was working before or if this file is unused